src/package/nuspec/Microsoft.TestPlatform.TestHost.NetCore.props

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Condition=" '$(Platform)' == 'x86'">
```

Comment:
You can check with $(OS) property to selectively drop these exe's, this will also help while fetching testhost path in runtime provider.